lambdas/catalog-updater/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "aws-serverless-dev-portal-example",
```

Comment:
name and description should be updated to accurately reflect what this lambda fn is doing

Reply:
Done!